crates/extensions/c8y_mapper_ext/src/supported_operations/mod.rs

```diff
@@ -226,6 +228,27 @@ impl SupportedOperations {
         Ok(device_xid)
     }
 
+    /// Get child device external IDs from subdirectory names.
+    pub fn get_child_xids(&self) -> Vec<ExternalId> {
```

Comment:
I'm surprised to see yet another way for the mapper to get a list of child devices.

Reply:
Good point. The requirement here is to get the external IDs of all child devices and services, including nested ones. I'll try to use EntityStore instead.

Actually it reminds of me that I forgot to add a test against service.

Reply:
Indeed fixed, but here https://github.com/thin-edge/thin-edge.io/pull/3761/commits/0525623ba7fbcc11775ca3552ed7a1329c3bcc64